docs/user/use-ebrains.rst

```diff
@@ -0,0 +1,40 @@
+Use NEST Desktop on EBRAINS
+===========================
+
+
+.. image:: ../_static/img/logo/ebrains-logo.svg
+  :width: 320px
+  :alt: EBRAINS
+
+|
+
+**EBRAINS** is a platform infrastructure for neuroscience.
+NEST Desktop is available as a prototype online.
+
+.. raw:: html
+
+  <div class="iframe-container">
+    <iframe src="https://ebrains.eu/service/nest-desktop" frameborder="0" allowfullscreen></iframe>
+  </div>
+
+|
+
+.. note::
+
+  You need an **EBRAINS** account to access NEST Desktop.
+
+
+||||
+
+**Trouble shootings**
+
+Why cannot I find NEST Server?
+  Sometimes the issue is resolved when you check NEST Server.
+
+  If not, the problem lies in cookies and site data for the user authentication of the *EBRAINS*.
+  That cookie expires after a session.
+  This happens when you re-access to ``https://nest-desktop.apps.hbp.eu`` after the browser is closed.
+
+  The simple solution is to reload the page (:guilabel:`CTRL` + :guilabel:`SHIFT` + :guilabel:`R`) that you can re-login.
```

Comment:
```suggestion
  A simple solution is to reload the page with deleting the cookies (:guilabel:`CTRL` + :guilabel:`SHIFT` + :guilabel:`R`) so that you can re-login.
```